src/Symfony/Component/Messenger/Tests/FailureIntegrationTest.php

```diff
@@ -208,7 +205,8 @@ public function testRequeMechanism()
          * Dispatch the original message again
          */
         $bus->dispatch($envelope);
-        // handle the message, but with no retries
+        // handle the failing message so it goes into the failure transport
+        $runWorker('transport1');
```

Comment:
extra `$runWorker()` needed now?

Reply:
yes to make the message fail + retry so that it goes into the failure transport. the test was failing before which I didn't because of the broken repo split at the moment